onnxruntime/core/providers/webnn/builders/model.cc

```diff
@@ -178,6 +183,9 @@ onnxruntime::common::Status Model::Dispatch(const InlinedHashMap<std::string, On
     promises.call<void>("push", ml_tensor);
   }
   auto ml_tensors = emscripten::val::global("Promise").call<emscripten::val>("all", promises).await();
```

Comment:
I'd suggest just adding the `console.time` before this line, `Promise.all` is the major time consumer for constructing the ml tensors.

Reply:
We can get a high level view of time spending before wasm Dispatch API call if Promise.all is included.